diff --color -ur layout/layout.ejs layout-new/layout.ejs
--- layout/layout.ejs	2021-05-23 19:13:20.000000000 +0800
+++ layout-new/layout.ejs	2021-06-26 20:31:37.208463200 +0800
@@ -66,7 +66,7 @@
     <% } %>
 
     <% if (theme.scroll_top_arrow.enable) { %>
-      <a id="scroll-top-button" href="#" role="button">
+      <a id="scroll-top-button" aria-label="TOP" href="#" role="button">
         <i class="iconfont icon-arrowup" aria-hidden="true"></i>
       </a>
     <% } %>
diff --color -ur layout/_partial/head.ejs layout-new/_partial/head.ejs
--- layout/_partial/head.ejs	2021-05-23 19:13:20.000000000 +0800
+++ layout-new/_partial/head.ejs	2021-06-26 20:44:33.923439500 +0800
@@ -18,7 +18,7 @@
   <link rel="apple-touch-icon" sizes="76x76" href="<%- url_for(theme.apple_touch_icon) %>">
   <link rel="icon" href="<%- url_for(theme.favicon) %>">
   <meta name="viewport"
-        content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, shrink-to-fit=no">
+        content="width=device-width, initial-scale=1.0, maximum-scale=5.0, shrink-to-fit=no">
   <meta http-equiv="x-ua-compatible" content="ie=edge">
   <% if (theme.force_https) { %>
     <meta http-equiv="Content-Security-Policy" content="upgrade-insecure-requests">
diff --color -ur layout/_partial/nav.ejs layout-new/_partial/nav.ejs
--- layout/_partial/nav.ejs	2021-05-23 19:13:20.000000000 +0800
+++ layout-new/_partial/nav.ejs	2021-06-26 20:32:06.267759000 +0800
@@ -20,7 +20,7 @@
           } %>
           <% if (each.submenu) { %>
             <li class="nav-item dropdown">
-              <a class="nav-link dropdown-toggle" target="_self" role="button"
+              <a class="nav-link dropdown-toggle" target="_self" href="javascript:;" role="button"
                  data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                 <%- each.icon ? '<i class="' + each.icon + '"></i>' : '' %>
                 <%- text %>
@@ -50,13 +50,13 @@
         <% } %>
         <% if(theme.search.enable) { %>
           <li class="nav-item" id="search-btn">
-            <a class="nav-link" target="_self" data-toggle="modal" data-target="#modalSearch">&nbsp;<i
+            <a class="nav-link" target="_self" href="javascript:;" data-toggle="modal" data-target="#modalSearch" aria-label="Search">&nbsp;<i
                 class="iconfont icon-search"></i>&nbsp;</a>
           </li>
         <% } %>
         <% if(theme.dark_mode && theme.dark_mode.enable) { %>
           <li class="nav-item" id="color-toggle-btn">
-            <a class="nav-link" target="_self">&nbsp;<i
+            <a class="nav-link" target="_self" href="javascript:;" aria-label="Color Toggle">&nbsp;<i
                 class="iconfont icon-dark" id="color-toggle-icon"></i>&nbsp;</a>
           </li>
         <% } %>
